test/import/FSHImporter.Extension.test.ts

```diff
@@ -161,5 +162,20 @@ describe('FSHImporter', () => {
         assertOnlyRule(extension.rules[0], 'value[x]', { type: 'Quantity' });
       });
     });
+
+    // Since Extensions use the same rule parsing code as Profiles, only do a minimal tests
```

Comment:
Should be "minimal test" (singular), since there's only one `CaretValueRule` test. Actually, I see that this line appears repeatedly in `FSHImporter.Extension.test.ts`, and it would probably be fine to just have it once before all the tests that overlap with the Profile tests.

Reply:
Yeah fair point it's repetitive. I made it one comment at the top that is more grammatically sound.